backend/benefit/applications/services/applications_csv_report.py

```diff
@@ -193,13 +204,44 @@ def CSV_COLUMNS(self):
                 format_bool,
                 default_value=None,
             ),
+            CsvDefaultColumn(
+                "Hyväksymisen/hylkäyksen/peruutuksen syy", "latest_decision_comment"
+            ),
             CsvDefaultColumn("Päättäjän nimike", "batch.decision_maker_title"),
             CsvDefaultColumn("Päättäjän nimi", "batch.decision_maker_name"),
             CsvDefaultColumn("Päätöspykälä", "batch.section_of_the_law"),
             CsvDefaultColumn("Päätöspäivä", "batch.decision_date"),
             CsvDefaultColumn("Asiantarkastajan nimi", "batch.expert_inspector_name"),
             CsvDefaultColumn("Asiantarkastajan email", "batch.expert_inspector_email"),
+            # In case there are multiple rows per application, always have the nth ahjo row
+            # in the same column.
+            # The row data here comes from calculation.ahjo_rows[application_row_idx]
```

Comment:
Is it so that application_row_idx starts from 1 and not from 0? Looking at the getter function that would make sense. The comment here is slightly misleading or at least a bit ambiguous because it's referring to dereferencing an array with the ahjo_rows[application_row_idx] instead of ahjo_rows[application_row_idx - 1]. Maybe a comment here and/or in the getter function about the application_row_idx starting from 1 rather than from 0 and fixing the used index here would do the trick?

Reply:
fixed comment